homeassistant/components/hunterdouglas_powerview/sensor.py

```diff
@@ -127,6 +130,7 @@ def __init__(
         self._attr_name = f"{self._shade_name} {description.name}"
         self._attr_unique_id = f"{self._attr_unique_id}_{description.key}"
         self._attr_native_unit_of_measurement = description.native_unit_of_measurement
+        self._attr_entity_registry_visible_default = description.visible_default
```

Comment:
We normally don't hide entities unless they are represented by another entity, eg a group. Is this really what is appropriate here?

It's more common to disable non useful entities.

Reply:
set to disabled now